src/main/java/edu/harvard/iq/dataverse/FileMetadata.java

```diff
@@ -472,6 +472,22 @@ public void setSelected(boolean selected) {
         this.selected = selected;
     }
     
+    /*
+    For use during file upload so that the user may delete 
+    files that have already been uploaded to the current dataset version
+    */
+    
+    @Transient
+    private boolean markedAsDuplicate;
+
+    public boolean isMarkedAsDuplicate() {
```

Comment:
is this for checksums? if so, do we need, since we already have it on the datafile object?

Reply:
We actually do need it because there's a separate list of file metadata that feed the uploaded files table and if the user deletes the duplicates we have to remove them too. I was also using it to write the success message, but since were no longer using names there I did some code simplification there.

Reply:
But can't you call fileMetaData.datafile.isMarkedAsDuplicate()? or something like that?